### PR TITLE
Updated docs with required field & optional value

### PR DESCRIPTION
**Pull request by harshit-splunk**

### CODE REVIEW

**Review by luckyj5 — APPROVED** [review]

**Comment by luckyj5** [comment]
LGTM. thanks

File: README.md [hunk 1]
```diff
@@ -57,45 +57,45 @@ or later. Earlier versions should use `cloud_controller.admin` instead.
...
+* `EXTRA_FIELDS`: Extra fields to annotate your events with (format is key:value,key:value). (Default: "")
+* `FLUSH_INTERVAL`: Time interval (in s/m/h. For example, 3600s or 60m or 1h) for flushing queue to Splunk regardless of CONSUMER_QUEUE_SIZE. Protects against stale events in low throughput systems. (Default: 5s)
+* `CONSUMER_QUEUE_SIZE`: Sets the internal consumer queue buffer size. Events will be pushed to Splunk after queue is full. (Default: 10000)
+* `HEC_BATCH_SIZE`: Set the batch size for the events to push to HEC (Splunk HTTP Event Collector). (Default: 100)
+* `HEC_RETRIES`: Retry count for sending events to Splunk. After expiring, events will begin dropping causing data loss. (Default: 5)
+* `HEC_WORKERS`: Set the amount of Splunk HEC workers to increase concurrency while ingesting in Splunk. (Default: 8)
+* `ENABLE_EVENT_TRACING`: Enables event trace logging. Splunk events will now contain a UUID, Splunk Nozzle Event Counts, and a Subscription-ID for Splunk correlation searches. (Default: false)
+* `SPLUNK_VERSION`: The Splunk version that determines how HEC ingests metadata fields. Only required for Splunk version 6.3 or below. (Default: "7.2")
```
Comment (luckyj5) [inline README.md:98]: let's remove this. 6.3 is old  and wont be needed anymore. Also, if there is any ref of this in the code, lets clean that up as well.

File: README.md [hunk 2]
```diff
@@ -57,45 +57,45 @@ or later. Earlier versions should use `cloud_controller.admin` instead.
...
+* `ADD_TAGS`: Add additional tags from envelope to splunk event. (Default: false)
+* `IGNORE_MISSING_APP`: If the application is missing, then stop repeatedly querying application info from Cloud Foundry. (Default: true)
+* `MISSING_APP_CACHE_INVALIDATE_TTL`:  How frequently the missing app info cache invalidates (in s/m/h. For example, 3600s or 60m or 1h). (Default: 0s)
+* `APP_CACHE_INVALIDATE_TTL`: How frequently the app info local cache invalidates (in s/m/h. For example, 3600s or 60m or 1h). (Default: 0s)
+* `ORG_SPACE_CACHE_INVALIDATE_TTL`: How frequently the org and space cache invalidates (in s/m/h. For example, 3600s or 60m or 1h). (Default: 72h)
+* `APP_LIMITS`: Restrict to APP_LIMITS the most updated apps per request when populating the app metadata cache. keep it 0 to update all the apps. (Default: 0)
+* `BOLTDB_PATH`: Bolt database path. (Default: cache.db)
+* `EVENTS`: A comma separated list of events to include. Possible values: ValueMetric,CounterEvent,Error,LogMessage,HttpStartStop,ContainerMetric. If it is blank, nozzle will automatically select LogMessage (Default: "ValueMetric,CounterEvent,ContainerMetric")
```
Comment (luckyj5) [inline README.md:90]: EVENTS - It's a required field and cannot be blank in the Tile. 
Also, is the default "logmessage" or "Default: "ValueMetric,CounterEvent,ContainerMetric"

Comment (harshit-splunk) [reply README.md:90]: We are required to select at least  one eventtype in the tile.
But in local execution or pushed via cf, if we explicitly set EVENTS="", nozzle will automatically select logmessage at the parsing level [parsing level](https://github.com/cloudfoundry-community/splunk-firehose-nozzle/blob/edd2401b7a787a1f59c19d619f07c4a258b11c68/events/events.go#L283).

**Review by luckyj5 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**